pkg/utils/utils.go

```diff
@@ -60,6 +63,29 @@ func GetPfAddr(pciAddr string) (string, error) {
 // GetPfName returns SRIOV PF name for the given VF
 // If device is not VF then it will return its own ifname if exist else empty string
 func GetPfName(pciAddr string) (string, error) {
+	pfEswitchMode, err := GetPfEswitchMode(pciAddr)
+	if err != nil {
+		errString := strings.ToLower(fmt.Sprint(err))
+		// If device doesn't support eswitch mode query or doesn't have sriov enabled,
+		// fall back to the default implementation
+		if strings.Contains(errString, "no such device") {
```

Comment:
can we do:

```
if strings.Contains(errString, "no such device") || strings.Contains(errString, "operation not supported" {
  glog.Infof("devlink query for eswitch mode is not supported for device %s. %v", pciAddr, err)
} else {
  return "", err
}
...
```

as its not really a warning, we might have devices that simply dont support devlink.